buster/docparser.py

```diff
@@ -75,46 +76,54 @@ def get_file_extension(filepath: str) -> str:
     return os.path.splitext(filepath)[1]
 
 
-def write_documents(filepath: str, documents_df: pd.DataFrame):
+def write_documents(filepath: str, source: str, documents_df: pd.DataFrame):
```

Comment:
should source be an optional arg? Seems to only be used in the case where `ext == ".db"`

Reply:
the problem is that it is very much not optional when using the db. write/read documents need to be removed completely, and redefined as an interface

Reply:
i'm still not sure i follow, isn't this behaviour the opposite of what we want? Right now, the `Chatbot` instance expects the `name` and `text` fields, but you're naming them to `title` and `content`?

Reply:
oh i see... ignore my previous comment

Reply:
this seems a little confusing, and while i agree with redefining our naming convention, i dont think i agree with doing it half-half with a TODO, couldn't we just grep back all the current `title` and `content` in this pr to `name` and `text`, add the TODO, and do a PR right after changing them all properly? We will also have to change all the fields in the dataframes we've already generated

Reply:
I agree half half is not great. I'd prefer going all the way in this PR then, rather than undo my work now, then redo it right after